### PR TITLE
Fixes #12111 - Set width for host list action column

### PR DESCRIPTION
**Pull request by tbrisker**
This makes no siginificant change to core but allows unbreaking
foreman_column_view plugin.

### CODE REVIEW

**Comment by dLobatog** [comment]
@tbrisker I'm getting `ActionView::Template::Error (couldn't find file 'bootstrap' with type 'application/javascript'` when trying the column_view PR , any idea what's wrong with it? I don't think it's a local setup issue because when I remove the plugin from my Gemfile I don't see the issue anymore.

File: app/views/hosts/_list.html.erb [hunk 1]
```diff
@@ -10,7 +10,7 @@
       <th class="hidden-tablet hidden-xs" width="10%"><%= sort :model, :as => _('Model') %></th>
       <th class="hidden-tablet hidden-xs" width="15%"><%= sort :hostgroup, :as => _("Host group") %></th>
       <th class="hidden-tablet hidden-xs" width="10%"><%= sort :last_report, :as => _('Last report'), :default => 'DESC' %></th>
-      <th ><%= _('Actions') %></th>
+      <th width="100px"><%= _('Actions') %></th>
```
Comment (dLobatog) [inline app/views/hosts/_list.html.erb:13]: What you want to accomplish by using fixed-size width is that other buttons never cause this one to overflow, right?

Comment (tbrisker) [reply app/views/hosts/_list.html.erb:13]: That is correct.

**Comment by tbrisker** [comment]
@dLobatog i'm not quite sure where you would be getting that, there is no mention of bootstrap anywhere in the column view plugin. Is it possible you may have something in your `settings.yaml` that causes it? or maybe an older version of the plugin? it seems to work just fine for me with a git checkout.

**Comment by dLobatog** [comment]
Just saw what caused it, it's https://github.com/GregSutcliffe/foreman_column_view/pull/23 . This fixes the issue in the plugin and does not cause problems here so :+1:

**Comment by dLobatog** [comment]
Thanks  @tbrisker !

**Pull request merged**